.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
+
+jobs:
+  tests: 
+    parameters:
+      python-version:
+        type: string 
+    docker:
+      - image: circleci/python:<< parameters.python-version >>
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: pip install poetry && poetry config virtualenvs.create false && poetry install 
+      - run:
+          name: Run test
+          command: invoke test
+
+workflows:
+  pasar-tests: 
+    jobs:
+      - tests:
+          matrix:
+            parameters: 
+              python-version: ["3.7", "3.8", "3.10"]
```

Comment:
aunque el proyecto lo estés desarrollando tu en la versión 3.9 puede algun usuario pedirte PR y aportar su trocito usando otra versión de python. Incluso tu mismo durante la vida del proyecto puedes actualizar la versión de python de tu ordenador y desarrollarlo en otra.
Con esto quiero decir que no es mala idea la de en la matriz incluir la versión 3.9 también

Reply:
La version 3.9 es la que se usa en los test y por lo tanto no creo que sea necesario ya que comprobarías dos veces la versión 3.9

Reply:
> La version 3.9 es la que se usa en los test y por lo tanto no creo que sea necesario ya que comprobarías dos veces la versión 3.9

¿En qué tests?
En general, conviene que le hagáis caso a las personas que os revisan. También "es la que uso en x" no es un argumento en contra de lo que dice @lentes4k